src/network/networkd-link.c

```diff
@@ -154,6 +154,11 @@ bool link_is_ready_to_configure(Link *link, bool allow_unmanaged) {
         if (link->set_link_messages > 0)
                 return false;
 
+        /* TODO: enable this check when link_request_to_create_stacked_netdev() is used.
```

Comment:
doesn't matter, but our usual coding style suggests using // comments for FIXME/TODO style comments

Reply:
OK. But this comment is already dropped in later commit. Hence, keep this as is.